src/components/AccountConnection/AccountConnection.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../styles/foundation';
```

Comment:
I _reeeaally_ don't like this being a relative path. Developers should not have to fight with SASS errors / failed builds as they in/decrement `../` trying to get to the right path.

Why are these not all absolute?

Reply:
See [my comment below](https://github.com/Shopify/polaris-react/pull/1680#issuecomment-501908524) – I'd agree with you in an app context, but when building a library there is a case for optimizing the consumer experience, which sometimes comes at the expense of the contributor's experience.